library/cipher_wrap.h

```diff
@@ -93,9 +93,11 @@ struct mbedtls_cipher_base_t {
     int (*setkey_enc_func)(void *ctx, const unsigned char *key,
                            unsigned int key_bitlen);
 
+#if !defined(MBEDTLS_BLOCK_CIPHER_NO_DECRYPT)
```

Comment:
If we want to make the option compatible with DES (with no effect on it), we'll need to change that to `#if DES || !NO_DECRYPT`.